modoboa_amavis/sql_connector.py

```diff
@@ -215,22 +213,11 @@ def get_pending_requests(self):
 
     def get_mail_content(self, mailid):
         """Retrieve the content of a message."""
-        content = b"".join([
+        return b"".join([
             smart_bytes(qmail.mail_text)
             for qmail in Quarantine.objects.filter(
                     mail=smart_bytes(mailid))
         ])
-        try:
-            content = content.decode("utf-8")
-        except UnicodeDecodeError:
-            pass
-        else:
-            return content
-        try:
-            result = chardet.detect(content)
```

Comment:
Are you sure this is not required anymore? Some emails lie about their encoding...

Reply:
correct, especially considering in this context they may be coming from an untrustworthy source, code restored.

I've merged modoboa/modoboa#1347 to remove an assert that was used for testing that would break this code.